java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -0,0 +1,456 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands.servermodules;
+
+import static glide.api.models.GlideString.gs;
+
+import glide.api.BaseClient;
+import glide.api.GlideClient;
+import glide.api.GlideClusterClient;
+import glide.api.models.ClusterValue;
+import glide.api.models.GlideString;
+import glide.api.models.commands.ConditionalChange;
+import glide.api.models.commands.json.JsonGetOptions;
+import glide.api.models.commands.json.JsonGetOptionsBinary;
+import glide.utils.ArgsBuilder;
+import glide.utils.ArrayTransformUtils;
+import java.util.concurrent.CompletableFuture;
+import lombok.NonNull;
+
+/** Module for JSON commands. */
+public class Json {
+
+    public static final String JSON_PREFIX = "JSON.";
+    public static final String JSON_SET = JSON_PREFIX + "SET";
+    public static final String JSON_GET = JSON_PREFIX + "GET";
+
+    private Json() {}
+
+    /**
+     * Sets the JSON value at the specified <code>path</code> stored at <code>key</code>.
+     *
+     * @param client The Valkey GLIDE client to execute the command.
+     * @param key The <code>key</code> of the JSON document.
+     * @param path Represents the path within the JSON document where the value will be set. The key
+     *     will be modified only if <code>value</code> is added as the last child in the specified
+     *     <code>path</code>, or if the specified <code>path</code> acts as the parent of a new child
+     *     being added.
+     * @param value The value to set at the specific path, in JSON formatted string.
+     * @return A simple <code>"OK"</code> response if the value is successfully set.
+     * @example
+     *     <pre>{@code
+     * String value = Json.set(client, "doc", , ".", "{'a': 1.0, 'b': 2}").get();
+     * assert value.equals("OK");
+     * }</pre>
+     */
+    public static CompletableFuture<String> set(
+            @NonNull BaseClient client,
+            @NonNull String key,
+            @NonNull String path,
+            @NonNull String value) {
+        return executeCommand(client, new String[] {JSON_SET, key, path, value});
```

Comment:
I'd do this way. In case if you need to fix something - you'll fix it in a single place.
```suggestion
        return set(client, gs(key), gs(path), gs(value));
```

Reply:
String and GlideString versions return different types.